avaframe/out3SimpPlot/outPlotAllPeak.py

```diff
@@ -0,0 +1,71 @@
+"""
+
+This is a simple function for a quick plot of all peak files at once
+
+This file is part of Avaframe.
+
+"""
+
+import os
+import numpy as np
+import copy
+from matplotlib import pyplot as plt
+import seaborn as sns
+from avaframe.out3Plot.plotUtils import *
+from avaframe.in3Utils import fileHandlerUtils as fU
+import avaframe.out3Plot.makePalette as makePalette
+
+def plotAllPeakFields(avaDir, cfg):
+    """ Plot all peak fields and return dictionary with paths to plots """
+
+    # Load all infos on simulations
+    inputDir = os.path.join(avaDir, 'Outputs', 'com1DFA', 'peakFiles')
+    peakFiles = fU.makeSimDict(inputDir)
+
+    # Output directory
+    outDir = os.path.join(avaDir, 'Outputs', 'out3SimpPlot')
+    fU.makeADir(outDir)
+
+    # Initialise plot dictionary with simulation names
+    plotDict = {}
+    for sName in peakFiles['simName']:
+        plotDict[sName] = {}
+
+    # Loop through peakFiles and generate plot
+    for m in range(len(peakFiles['names'])):
+
+        # Load names and paths of peakFiles
+        name = peakFiles['names'][m]
+        fileName = peakFiles['files'][m]
+        log.info('now plot %s:' % (fileName))
+
+        # Load data
+        data = np.loadtxt(fileName, skiprows=6)
+        unit = cfg['REP']['unit%s' % peakFiles['resType'][m]]
+
+        # Set extent of peak file
+        cellSize = peakFiles['cellSize'][m]
+        ny = data.shape[0]
+        nx = data.shape[1]
+        Ly = ny*cellSize
+        Lx = nx*cellSize
+
+        # Figure  shows the result parameter data
+        fig = plt.figure(figsize=(figW, figH))
+        fig, ax1 = plt.subplots()
+        # choose colormap
+        # cmapDict = {'ppr' : cmapPres, 'pfd' : cmapDepth, 'pv' : cmapSpeed}
+        cmap, _, _, norm, ticks = makePalette.makeColorMap(
+            cmapPres, np.amin(data), np.amax(data), continuous=contCmap)
+        im1 = ax1.imshow(data, cmap=cmap, extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
+        addColorBar(im1, ax1, ticks, unit)
+        title = str('%s' % name)
+        ax1.set_title(title)
+        ax1.set_xlabel('x [m]')
+        ax1.set_ylabel('y [m]')
+        plotName = os.path.join(outDir, '%s.%s' % (name, outputFormat))
+        fig.savefig(plotName)
```

Comment:
Could you use the saveAndOrPlot function from plotUtils -> to honor global flags about saving/plotting...

Reply:
the cfgPaths variable is not set globally so i would move this as a request for a future PR?

Reply:
jup, move it to an issue